conanfile.py

```diff
@@ -9,7 +9,8 @@ class Clio(ConanFile):
     description = 'Clio RPC server'
     settings = 'os', 'compiler', 'build_type', 'arch'
     options = {
-        'fPIC': [True, False],
+        'static': [True, False],    # static linkage
+        'fPIC': [True, False],      # unused?
```

Comment:
We could remove this but i'm not 100% sure whether this is used by some deps or what. 
Or we could add `-fpic` flag to the compiler in case this is set.